commit 3f20a2bf9225a0103f918f81755613ab6d339eeb
Author: q66 <q66@chimera-linux.org>
Date:   Sat Jul 20 14:07:44 2024 +0200

    disable *printf fortify stuff
    
    This has known issues; under clang all the actually variadic
    ones are disabled by default but some are still enabled.

diff --git a/include/stdio.h b/include/stdio.h
index 642ac40..9b2d7c2 100644
--- a/include/stdio.h
+++ b/include/stdio.h
@@ -33,8 +33,10 @@
 #undef fopen
 #undef fread
 #undef fwrite
+#if 0
 #undef vsnprintf
 #undef vsprintf
+#endif
 
 #if defined(_POSIX_SOURCE) || defined(_POSIX_C_SOURCE) \
  || defined(_XOPEN_SOURCE) || defined(_GNU_SOURCE) \
@@ -143,6 +145,7 @@
 }
 #endif
 
+#if 0
 __fh_access(write_only, 1, 2)
 __fh_access(read_only, 3)
 __fh_format(printf, 3, 0)
@@ -188,8 +191,9 @@
 	return __r;
 #endif
 }
+#endif
 
-#ifndef __clang__  /* FIXME */
+#if 0
 #undef vfprintf
 #undef vprintf
 
diff --git a/tests/Makefile b/tests/Makefile
index 56455bd..bfc7868 100644
--- a/tests/Makefile
+++ b/tests/Makefile
@@ -130,9 +130,6 @@ RUNTIME_TARGETS= \
 	test_vasprintf               \
 	test_vfprintf                \
 	test_vprintf                 \
-	test_vsnprintf_dynamic       \
-	test_vsnprintf_static        \
-	test_vsprintf                \
 	test_wcscat_static_write     \
 	test_wcscpy_static_write     \
 	test_wcsncat_static_write    \
